java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/build/LookupBuildInfoCommand.java

```diff
@@ -133,19 +127,10 @@ public void run() {
 
             Log.infof("Cloning commit %s (tag %s) for path %s", commit, tag, context);
             doBuildAnalysis(info.getUriWithoutFragment(), commit, context, buildRecipeInfo, privateRepo, buildInfoLocator);
-
-            if (message != null) {
-                Files.createFile(message);
-            }
         } catch (Exception e) {
             Log.errorf(e, "Failed to process build info for " + scmUrl);
-            if (message != null) {
-                try {
-                    Files.writeString(message, "Failed to analyse build for " + scmUrl + ". Failure reason: " + e.getMessage());
-                } catch (IOException ex) {
-                    Log.errorf(e, "Failed to write result");
-                }
-            }
+            resultsUpdater.get().updateResults(taskRun, Map.of(
```

Comment:
Shouldn't this be the message rather than BUILD_INFO? The build info should be JSON

Reply:
Ah, I'd trimmed it down so in the error case, it just passed the string back through build_info and processed it 
 https://github.com/redhat-appstudio/jvm-build-service/pull/807/files#diff-1dcd0d035636ba6f1cb8632a7ea1820f33bca4ecdc6a118dfb0f67b0734498f3R309